DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/ConfirmDeleteGroup.cshtml

```diff
@@ -16,11 +16,11 @@
       <vc:error-summary order-of-property-names="@(new[] { nameof(Model.Confirm), nameof(Model.DeleteEnrolments) })" />
     }
 
-    <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-8">Delete delegate group</h1>
+    <h1 class="nhsuk-heading-xl">Delete delegate group</h1>
   </div>
 </div>
 
-<div class="nhsuk-grid-row">
+<div class="nhsuk-grid-row nhsuk-u-margin-bottom-8">

```

Comment:
There seems to be slight variation on the markup and css in these Delete/Remove" type screens. Not that we have too many of them.
Of the newer ones, it would be good to be consistent, for that reason I would say drop `nhsuk-u-margin-bottom-8` from this line.